media/js/firefox/welcome/welcome14.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+(function () {
+    'use strict';
+
+    var copyButton = document.getElementById('code-copy');
```

Comment:
Nit: prefer `const` over `var`
```suggestion
    const copyButton = document.getElementById('code-copy');
```